src/main/webapp/Portal.html

```diff
@@ -504,9 +509,8 @@
       head.load(cdnEndpoint + 'generated/prop-types.min.js')
       head.load(cdnEndpoint + 'generated/react-measure.js')
       head.load(cdnEndpoint + 'generated/plotly.min.js')
-      head.load(
-        'https://unpkg.com/react-plotly.js@2.2.0/dist/create-plotly-component.min.js',
-      )
+      head.load(cdnEndpoint + 'js/2.2.0_create-plotly-component.min.js')
+      head.load(cdnEndpoint + 'generated/react-jsonschema-form.js')
```

Comment:
react-jsonschema-form should stay removed from the list of scripts to load, no?

Can we add react-plotly.js to the package.json and add build actions to copy the react-plotly.js dependency like we do for the other scripts (to get Dependabot parsing of our package.json/yarn.lock files?)

Reply:
And you're absolutely right regarding create-plotly-component.min.js from react-plotly.  This is not a special case